hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -245,6 +257,8 @@ public Response put(
         throw newError(S3ErrorTable.ACCESS_DENIED, keyPath, ex);
       } else if (ex.getResult() == ResultCodes.BUCKET_NOT_FOUND) {
         throw newError(S3ErrorTable.NO_SUCH_BUCKET, bucketName, ex);
+      } else if (ResultCodes.FILE_ALREADY_EXISTS == ex.getResult()) {
```

Comment:
nit. to match formatting
```suggestion
      } else if (ex.getResult() == ResultCodes.FILE_ALREADY_EXISTS) {
```